control-plane-operator/controllers/hostedcontrolplane/csi/kubevirt/files/tenant_controller_clusterrole.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubevirt-csi-controller-cr
+rules:
+  # Allow listing and creating CRDs
```

Comment:
which new CRDs are needed?

Reply:
they aren't anymore. I removed this and fixed it upstream as well.